app/DynamicsAdapter/Fams3Adapter.Dynamics.Test/SearchApiRequest/SearchApiRequestServiceTest.cs

```diff
@@ -29,16 +29,50 @@ public void SetUp()
             int readyForSearchVAlue = SearchApiRequestStatusReason.ReadyForSearch.Value;
             int inProgressValue = SearchApiRequestStatusReason.InProgress.Value;
 
+            //odataClientMock.Setup(x => x.For<SSG_SearchApiRequest>(null)
```

Comment:
Remove commented code ;-)

Reply:
good catch. sorry to forget to remove it.